cmd/influxd/run.go

```diff
@@ -662,10 +651,8 @@ func (cmd *RunCommand) openServer(joinURLs []url.URL) *influxdb.Server {
 	return s
 }
 
-// joins a server to an existing cluster.
-func joinServer(s *influxdb.Server, u url.URL, joinURLs []url.URL) {
-	// TODO: Use separate broker and data join urls.
-
+// joinOrInitializeServer a server to an existing cluster or initializes.
+func joinOrInitializeServer(s *influxdb.Server, u url.URL, joinURLs []url.URL) {
```

Comment:
Now that I see it I don't fully understand the comments in this function. 

"Create data node on an existing data node." -- what do this mean?